integration_test/README.md

```diff
@@ -57,15 +57,15 @@ against a pre-built but unreleased agent, you can use add the
 `AGENT_PACKAGES_IN_GCS` environment variable onto your command like this:
 
 ```
-AGENT_PACKAGES_IN_GCS=gs://ops-agents-public-buckets-test-logs/prod/stackdriver_agents/testing/consumer/ops_agent/build/buster/2068/20220926-132259/result \
+AGENT_PACKAGES_IN_GCS=gs://ops-agents-public-buckets-test-logs/prod/stackdriver_agents/testing/consumer/ops_agent/build/bullseye/2068/20220926-132259/result \
```

Comment:
Should probably point to the actual bucket (and the actual log URL) from a more recent build…

Reply:
Where do I get one of those?

Reply:
Here's one from this PR: https://btx.cloud.google.com/invocations/a9f55e14-d695-4335-9e9e-478e5f31c882/details. Note also that `bullseye` has been renamed to `bullseye_x86_64` since this doc was updated…

Reply:
[e18feca](https://github.com/GoogleCloudPlatform/ops-agent/pull/1752/commits/e18feca17e427d43c5d2ed801276a535b2105733)